src/NuGet.Clients/NuGet.VisualStudio.Implementation/GlobalSuppressions.cs

```diff
@@ -11,15 +11,13 @@
 [assembly: SuppressMessage("Build", "CA1822:Member GetRegistryRepositoryPath does not access instance data and can be marked as static (Shared in VisualBasic)", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.VisualStudio.PreinstalledPackageInstaller.GetRegistryRepositoryPath(System.String,System.Collections.Generic.IEnumerable{NuGet.VisualStudio.IRegistryKey},System.Action{System.String})~System.String")]
-[assembly: SuppressMessage("Build", "CA1801:Parameter packageInstaller of method PerformPackageInstallAsync is never used. Remove the parameter or use it in the method body.", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.VisualStudio.PreinstalledPackageInstaller.PerformPackageInstallAsync(NuGet.VisualStudio.IVsPackageInstaller,EnvDTE.Project,NuGet.VisualStudio.PreinstalledPackageConfiguration,System.Boolean,System.Action{System.String},System.Action{System.String})~System.Threading.Tasks.Task")]
 [assembly: SuppressMessage("Build", "CA1305:The behavior of 'string.Format(string, object, object)' could vary based on the current user's locale settings. Replace this call in 'PreinstalledPackageInstaller.PerformPackageInstallAsync(IVsPackageInstaller, Project, PreinstalledPackageConfiguration, bool, Action<string>, Action<string>)' with a call to 'string.Format(IFormatProvider, string, params object[])'.", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.VisualStudio.PreinstalledPackageInstaller.PerformPackageInstallAsync(NuGet.VisualStudio.IVsPackageInstaller,EnvDTE.Project,NuGet.VisualStudio.PreinstalledPackageConfiguration,System.Boolean,System.Action{System.String},System.Action{System.String})~System.Threading.Tasks.Task")]
-[assembly: SuppressMessage("Build", "CA1801:Parameter provider of method AddFromExtension is never used. Remove the parameter or use it in the method body.", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.VisualStudio.PreinstalledRepositoryProvider.AddFromExtension(NuGet.Protocol.Core.Types.ISourceRepositoryProvider,System.String)")]
+[assembly: SuppressMessage("Build", "CA1801:Parameter provider of method AddFromExtension is never used. Remove the parameter or use it in the method body.", Justification = "Shipped public API", Scope = "member", Target = "~M:NuGet.VisualStudio.PreinstalledRepositoryProvider.AddFromExtension(NuGet.Protocol.Core.Types.ISourceRepositoryProvider,System.String)")]
```

Comment:
unused parameter `provider` is already deleted from the code in this PR. https://github.com/NuGet/NuGet.Client/pull/4177/files#diff-a6da29e95db056fb704760cee1337bebd1c9f98f8263f27e04510aacc9702cb3R79. I think we can remove this suppression.

Reply:
Removed.